<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Presenter</class>
 <widget class="QWidget" name="Presenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Presenter</string>
  </property>
  <widget class="QListWidget" name="questions_list_widget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>531</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="id_line_edit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="text_line_edit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="answer_line_edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>530</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Text</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Answer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>add_button</sender>
   <signal>clicked()</signal>
   <receiver>Presenter</receiver>
   <slot>add_question()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>804</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_question()</slot>
 </slots>
</ui>
